lbplanner/settings.php

```diff
@@ -18,16 +18,43 @@
  * Defines some settings
  *
  * @package local_lbplanner
- * @copyright 2024 NecodeIT
+ * @copyright 2025 NecodeIT
  * @license https://creativecommons.org/licenses/by-nc-sa/4.0/ CC-BY-NC-SA 4.0 International or later
  */
 
+use local_lbplanner\enums\SETTINGS;
+
 defined('MOODLE_INTERNAL') || die;
 
 if ($hassiteconfig) {
-    $settings = new admin_settingpage('local_lbplanner', 'LB Planer');
+    $settings = new admin_settingpage('local_lbplanner', 'Eduplanner');
     $ADMIN->add('localplugins', $settings);
 
-    $settings->add(new admin_setting_configtext('local_lbplanner/activeyear',
-        'Active year', 'Current year  e.g.:"20/21" ', null, PARAM_TEXT));
+    $futuresightsett = new admin_setting_configselect(
+        'local_lbplanner/'.SETTINGS::SLOT_FUTURESIGHT,
+        'Slot Futuresight',
+        'How many days into the future students can reserve slots',
+        3,
+        [
+            0 => "0 Days",
+            1 => "1 Day",
+            2 => "2 Days",
+            3 => "3 Days",
+            4 => "4 Days",
+            5 => "5 Days",
+            6 => "6 Days",
+            7 => "7 Days",
+        ],
+    );
+    $settings->add($futuresightsett);
+
+    $outdaterangesett = new admin_setting_configduration(
+        'local_lbplanner/'.SETTINGS::COURSE_OUTDATERANGE,
+        'Course Outdaterange',
+        'How long after a course ends should it start being hidden in Eduplanner?',
```

Comment:
This should kick in regardless of the courses' end date. It should look at the date the course was created

Reply:
p.s. I didn't mean the stupid, sorry

Reply:
hmm fair enough. but to address the "thus can accumulate age theoretically indefinitely." wouldn't these courses also be hidden after 2 years for example? or is it possible to set an end date to `null` within a course thus rendering this option effectless for that particular course?

Reply:
it's possible to set the end date of a course to year 9999 or something like that. and you can also disable it, yeah